libs/langchain/langchain/__init__.py

```diff
@@ -62,10 +17,198 @@
 
 verbose: bool = False
 debug: bool = False
-llm_cache: Optional[BaseCache] = None
+llm_cache: Optional["BaseCache"] = None
+
+
+def __getattr__(name: str) -> Any:
+    warnings.warn(f"Importing {name} is no longer supported.")
```

Comment:
This should be reworded to something like "importing x from langchain root module is no longer supported, it should now be imported from langchain.x see more details at http://"

Reply:
what should i link to?